From 0a4e920ded688641b29bd7833a6e6c7673e9129e Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Thu, 10 Oct 2024 02:04:14 +0200
Subject: [PATCH] Use magick on Linux as well

WARNING: The convert command is deprecated in IMv7, use "magick" instead of "convert" or "magick convert"
---
 NiimPrintX/ui/component/FontList.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/NiimPrintX/ui/component/FontList.py b/NiimPrintX/ui/component/FontList.py
index f6d7a78f794e..e978a7300cbb 100644
--- a/NiimPrintX/ui/component/FontList.py
+++ b/NiimPrintX/ui/component/FontList.py
@@ -15,13 +15,10 @@ def fonts():
         elif platform.system() == "Windows":
             magick_path = os.path.join(imagemagick_base_path, 'magick.exe')
         elif platform.system() == "Linux":
-            magick_path = os.path.join(os.sep, 'usr', 'local', 'bin', 'convert')
+            magick_path = os.path.join(os.sep, 'usr', 'local', 'bin', 'magick')
         else:
             magick_path = 'magick'
     else:
-        if platform.system() == "Linux":
-            magick_path = 'convert'
-        else:
             magick_path = 'magick'
 
     # magick_path = 'imagemagick/bin/magick'
-- 
2.46.2

